torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12734,9 +12734,10 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         skips=(
             DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_small',
                          dtypes=(torch.int, torch.int8)),
-            # pytorch computes (0+nanj), numpy computes (-5e-18-1j) for input (-501.-1.0000e+20j)
-            DecorateInfo(unittest.expectedFailure, 'TestUnaryUfuncs',
-                         "test_reference_numerics_large", dtypes=(torch.complex64,)),),
+            # # pytorch computes (0+nanj), numpy computes (-5e-18-1j) for input (-501.-1.0000e+20j)
+            # DecorateInfo(unittest.expectedFailure, 'TestUnaryUfuncs',
+            #              "test_reference_numerics_large", dtypes=(torch.complex64,)),
```

Comment:
Just remove it, as this PR fixes this error :)

Reply:
Done. Thanks!